packages/react/src/themes/types.ts

```diff
@@ -92,18 +91,44 @@ export type PrimitiveColors = Partial<{
 type ExtendablePalette<T> = T &
   { [K in keyof T]?: K extends keyof PrimitiveColors ? string : ColorVariants }
 
-export type ColorPalette = ExtendablePalette<
-  EmphasisColorsStrict & ContextualColorsStrict & NaturalColorsStrict & PrimitiveColors
+export type ColorPalette<T = {}> = ExtendablePalette<
```

Comment:
Making the color palette extended - this typings are used only internally in the theme definition